File: Be/post-service/src/main/java/com/post/dto/response/TenantPostResponse.java
package com.post.dto.response;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

import java.math.BigDecimal;

@Getter
@Setter
@Builder
public class TenantPostResponse {

    String tenantPostId;

    String userId;

    String userName;

    String imgAvatar;

    String title;

    String description;

    long price;

    String location;

    Boolean isAvailable;

    String city;

    String district;

    String address;

    String ward;

    String tenantPostType;
}

File: Fe/client/src/dto/response/tenantPostResponse.ts
export type tenantPostResponse = {
	tenantPostId: string
	userId: string
	userName: string
	imgAvatar: string
	title: string
	description: string
	price: number
	location: string | null
	isAvailable: boolean
	city: string
	district: string
	address: string
	ward: string
	tenantPostType: string
}

export type tenantResponseOfLandlord = {
	userId: string;
	rentalDuration: string;
	startDate: string;
	endDate: string;
	isAvailable: boolean;
	roomTypeId: string;
	roomTypeName: string;
	apartmentId: string;
	apartmentName: string;
	roomName: string;
	roomId: string;
}
